.travis.yml

```diff
@@ -7,16 +7,24 @@ branches:
     - /^\d\.\d+\.\d+(rc\d+|\.dev\d+)?$/
 matrix:
   include:
+    - env: TOXENV=py 
+      python: 3.9-dev
+      dist: bionic
+    - env: TOXENV=extra-deps
+      python: 3.9-dev
+      dist: bionic
+    - env: TOXENV=asyncio
+      python: 3.9-dev
+      dist: bionic
```

Comment:
:lipstick: I would move these 3 at the end of the `include` list, separated by the 3.8 one with an empty line, in line with the existing list items.